### PR TITLE
[benchmark] fix combiner benchmarks

### PR DESCRIPTION
**Pull request by ehigham**
The combiner benchmarks broke following the deletion of the `experimental.vcf_combiner` python package. Re-implement them in terms of the `vds` package.

### CODE REVIEW

File: benchmark/python/benchmark_hail/__main__.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
+import sys
+
 from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
 
 from .run import cli as run
 from . import compare, create_resources, combine, summarize, visualize
 
-def main():
+def main(argv=sys.argv[1:]):
```
Comment (ehigham) [inline benchmark/python/benchmark_hail/__main__.py:8]: This change makes it easier for me to launch the benchmark suite in a debugger and change which benchmark i want to debug from within the debugger.

**Comment by ehigham** [comment]
Hi Chris, would you mind taking a look when you get a chance? I believe you might know the most about the functionality these are exercising. Do the new implementations look ok? Are they benchmarking equivalent functionality?

**Review by chrisvittal — APPROVED** [review]

File: benchmark/python/benchmark_hail/run/combiner_benchmarks.py [hunk 2]
```diff
@@ -43,56 +46,51 @@ def compile_2k_merge(path):
...
-                            use_exome_default_intervals=True)
+        combiner = new_combiner(
+            gvcf_paths=list(paths),
+            output_path=tmpdir,
+            temp_path='/tmp',
+            branch_factor=16,
+            reference_genome='GRCh38',
+            use_exome_default_intervals=True
+        )
+        combiner.run()
```
Comment (chrisvittal) [inline benchmark/python/benchmark_hail/run/combiner_benchmarks.py:81]: I believe this is now redundant.

**Review by chrisvittal — CHANGES_REQUESTED** [review]
This is indeed what we would like to be checking.

File: benchmark/python/benchmark_hail/run/combiner_benchmarks.py (hunk 2 above)
Comment (ehigham) [reply benchmark/python/benchmark_hail/run/combiner_benchmarks.py:81]: Great! I'll remove it. Thanks.

**Review by chrisvittal — APPROVED** [review]

**Pull request merged**